corehq/ex-submodules/dimagi/utils/csv.py

```diff
@@ -49,27 +49,15 @@ def __iter__(self):
 
 class UnicodeWriter:
```

Comment:
(non blocking) I don't think py3 catches this, but this is an "old style class" https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python I think it's supposed to be good practice to add `(object)`

Reply:
added trello card: https://trello.com/c/eKQrRDuR/62-use-new-style-classes

Reply:
https://github.com/dimagi/commcare-hq/pull/19582